rocketpool-cli/service/service.go

```diff
@@ -1055,6 +1043,16 @@ func pruneExecutionClient(c *cli.Context) error {
 
 	fmt.Printf("Your disk has %s free, which is enough to prune.\n", freeSpaceHuman)
 
+	if selectedEc == cfgtypes.ExecutionClient_Nethermind {
+		// Restarting NM is not needed anymore
+		err = rp.RunNethermindPruneStarter(executionContainerName)
+		if err != nil {
+			return fmt.Errorf("Error starting Nethermind prune starter: %w", err)
+		}
+		fmt.Printf("\nDone! Your main execution client is now pruning. You can follow its progress with `rocketpool service logs eth1`.\n")
+		fmt.Printf("%sNOTE: While pruning, you **cannot** interrupt the client (e.g. by restarting) or you risk corrupting the database!\nYou must let it run to completion!%s\n", colorYellow, colorReset)
```

Comment:
print these in `RunNethermindPruneStarter` on success.

The messaging here is slightly different than for geth, so it makes sense to copy it into the nethermind-specific function